docs/2_step_by_step/2_2_experiment_and_run.md

```diff
@@ -1,11 +1,11 @@
 (experiment-and-runs)=
 # Two basic concepts: Experiment and Run
 
-Finetuner organizes your training based on two concepts: `Experiment` and `Run`.
+Finetuner organizes your training based on two concepts: {class}`~finetuner.experiment.Experiment` and {class}`~finetuner.run.Run`.
```

Comment:
Here `Experiment` and `Run` are introduced as python classes, but the code snipped below doesn't actually use these classes. To me it is not quite clear, am I ever expected to use/instantiate these classes directly? If yes, I would say there needs to be an example. Or are they just something that ft uses under the hood and that I don't need to touch? In that case I would say don't link to the class reference, let's not make it part of the official API if it is not used by the user.

Reply:
this is correct, user in most cases, do not need to interact with `Experiment`, for run user need. For example, the return value of `.fit` is a `Run` object. I only linked their first occurence to the developer guide, let's see.